src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3159,10 +3096,7 @@ private bool IsFieldDeclaration(bool isEvent, bool isGlobalScriptLevel)
 
         private bool IsOperatorKeyword()
         {
-            return
-                this.CurrentToken.Kind == SyntaxKind.ImplicitKeyword ||
-                this.CurrentToken.Kind == SyntaxKind.ExplicitKeyword ||
-                this.CurrentToken.Kind == SyntaxKind.OperatorKeyword;
+            return this.CurrentToken.Kind is SyntaxKind.ImplicitKeyword or SyntaxKind.ExplicitKeyword or SyntaxKind.OperatorKeyword;
```

Comment:
patterns make kind checks much simpler (And faster to boot, though likely in a way that is utterly negligible). The readability here is much better though.